.github/.goreleaser-pre-release.yaml

```diff
@@ -0,0 +1,59 @@
+project_name: kubectl-storageos
+
+before:
+  hooks:
+    - go mod download
+
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+    goarch:
+      - amd64
+      - arm64
```

Comment:
Do we actually support arm64 anywhere?

Reply:
no, but I guess in theory you could manage a remote cluster from an arm device.. you just need this binary and a kubeconfig 😎